CONTRIBUTING.md

```diff
@@ -340,6 +340,19 @@ One way to accomplish this is by executing the following command:
 ./mvnw test -f integration-tests/resteasy-jackson/ -Dtest=GreetingResourceTest
 ```
 
+##### Maven Invoker tests
+
+For testing some scenarios, Quarkus uses the [Maven Invoker](https://maven.apache.org/shared/maven-invoker/) to run tests. For these cases, to run a single test, one needs to use the `invoker.test` property along with the name of the directory
+which houses the test project.
+
+For example, in order to only run the MySQL test project of the container-image tests, the Maven command would be:
+
+```
+./mvnw verify -f integration-tests/container-image/maven-invoker-way verify -Dinvoker.test=container-build-jib-with-mysql
```

Comment:
@geoand The doube-`verify` is probably a mistake as it yields:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-jar-plugin:3.1.2:jar (default-jar) on project quarkus-integration-test-container-image-invoker: You have to use a classifier to attach supplemental artifacts to the project instead of replacing them.
```
Also, this very specific module also requires `-Dstart-containers`.

That's what you get for being helpful by documenting things for others. 😉

Reply:
Ouch on the double `verify`... 

For the other flags, I was on the fence on whether to add them or not, because they don't bring any value to the specific section.

Feel free to update the doc how you best see fit :)

Reply:
If you are fixing that double-verify, you coud add one one-liner note on that this is just an example and that further parameters might be needed. 🤷

Reply:
Here it is: https://github.com/quarkusio/quarkus/pull/19651